tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -713,6 +715,20 @@ EOF
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+	if [ "${KATA_BUILD_CC}" == "yes" ]; then
```

Comment:
I don't think we should try and port `KATA_BUILD_CC` over to main. For pause image caching I think there are a couple of options:
1. Use the `GUEST_PULL` value to trigger this
2. Always include the pause bundle

I order to not risk decreasing performance of non-CC kata I think 1 is the way to go for now and then re-consider this later?

Reply:
Yes, I agree.